src/test/resources/data/FeedbackMsqQuestionUiTest.json

```diff
@@ -52,9 +52,57 @@
       "googleId": "FMsqQnUiT.alice.tmms",
       "email": "alice.b.tmms@gmail.tmt",
       "course": "FMsqQnUiT.CS2104",
-      "name": "Alice Betsy</td></div>'\"",
+      "name": "Alice Betsy",
       "comments": "This student's name is Alice Betsy</td></div>'\"",
-      "team": "Team </option><option value=\"dump\">1'"
+      "team": "Team 1"
```

Comment:
Why the need for the above two changes?

Reply:
I had trouble in selecting a checkbox in the submission page with `"team": "Team </option><option value=\"dump\">1'"`. The regular way to select an option was `toggleMsqOption()` of `FeedbackSubmitPage.java` which used `SanitizationHelper.sanitizeStringForXPath(choiceName)`. Somehow due to sanitization, I wasn't able to select the desired MSQ checkbox properly.

I looked at [code where the method was used](https://github.com/TEAMMATES/teammates/search?utf8=%E2%9C%93&q=toggleMsqOption&type=), in all of these cases, the name of the choice was simple, so I changed `Team </option><option value=\"dump\">1'` to `Team 1` to make the name simple, and henceforth I was able to select checkboxes properly.